examples/events/EiffelIssueDefinedEvent/simple.json

```diff
@@ -0,0 +1,24 @@
+{
+	"meta": {
+		"type": "EiffelIssueDefinedEvent",
+		"version": "1.0.0",
+		"time": 1234567890,
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0"
+	},
+	"data": {
+		"type": "BUG",
+		"tracker": "DONE", //the hell's a tracker?
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0",
+		"uri": "http://zombo.com/"
+	},
+	"links": [
+		{
+			"type": "BUG",
```

Comment:
`"BUG"` isn't a valid link type, we can get rid of it here.

Reply:
By get rid of, do you mean remove the line, or remove the block?

Reply:
The block.